CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Mike Dev
```

Comment:
2019

Reply:
Well, the file was created in 2018. But as the have been modifications in 2019 I probably should add that year.